Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:6.11-alpine
+RUN mkdir -p /code
+WORKDIR /code
+COPY package.json .
```

Comment:
I double checked why this way was recommended by the Node/Docker tutorial. It turns out that if we simply copy in all the code and then install deps, Docker will run the full dependency step every time _any_ file is changed in the . folder. 
Factoring out the dependency step and running it first will enable Docker to use its cache to run through the dependency step quickly if nothing has changed inside package.json. 
I did remove copying in the yarn.lock file though as the container will need its own version.